### PR TITLE
Validation SF 10_1_X

### PR DESCRIPTION
**Pull request by bcmcms**
Updated sampling factors for the run2 era and improved CaloTowers pedestal cuts to reflect QIE-11 thresholds. Additional single pion validation plots added by Salavat Abdoulline.

### CODE REVIEW

**Comment by hatakeyamak** [comment]
@deguio @mariadalfonso @spandeyehep @abdoulline

**Comment by abdoulline** [comment]
@bcmcms, @hatakeyamak    
please, fix the absence of EEHitCollection collection (for 2017/2018/run3)  here:  
https://github.com/cms-sw/cmssw/pull/22444/files#diff-cda29b4cec91d21c995171dd1f0cc3daR14
by explicitly removing it solely for phase2. Thank you.

**Comment by hatakeyamak** [comment]
@bcmcms had to add a few additional commits after the initial PR, but this should be now good to go from the developer side.

**Comment by davidlange6** [comment]
Your PR is unmergeable. Please have a look and possibly rebase it.

File: Validation/CaloTowers/test/macros/SinglePi.C [hunk 1]
```diff
@@ -297,23 +375,27 @@ void SinglePi(const TString ref_vers, const TString val_vers, bool fastsim){
...
-     f2_hist1[i]->Draw("histsames");   
+     f2_hist1[i]->Draw("hist sames");   
      
      leg->Draw();   
      
      myc->SaveAs(label1[i]);
 
      if(myc) delete myc;
+
+     std::cout << "1D histos " << i << " produced" << std::endl;
```
Comment (dmitrijus) [inline Validation/CaloTowers/test/macros/SinglePi.C]: Please remove this. Using std::cout is forbidden!

Comment (dmitrijus) [reply Validation/CaloTowers/test/macros/SinglePi.C]: Actually, this can be in, this is a test macro and no a framework module.

**Comment by dmitrijus** [comment]
+1

**Comment by fabiocos** [comment]
please test

**Comment by bcmcms** [comment]
After checking the test logs, I am having some trouble seeing what errors this PR produces, though I do see some internal messages with a severity of Error are tracked. I see that cmsbuild also gives the list of addon tests that failed as blank as well. Is there any more information available about what needs to be fixed on our end?

**Comment by fabiocos** [comment]
please test

**Comment by fabiocos** [comment]
@bcmcms are the observed differences in the SimHits consistent with what expected from the variation of the hf constants?

**Comment by bcmcms** [comment]
From our pion scans [here](http://hep.baylor.edu/cmshcal/mcmaster/10_postSOI/oldSF_vs_expSF_SinglePi/), we believe the variations within SimHits are expected and desired. We expect to see the overall behavior remain the same, but without spiking out of the valid energy range for |ieta|>=30. So far, these results seem to match that.

**Comment by fabiocos** [comment]
+1

**Pull request merged**